tests/in/glsl/math-functions.frag

```diff
@@ -44,7 +44,8 @@ void main() {
     vec4 minOut = min(a, b);
     vec4 reflectOut = reflect(a, b);
     vec3 crossOut = cross(a.xyz, b.xyz);
-    mat4 outerProductOut = outerProduct(a, b);
+    // TODO: support outerProduct function in wgsl output
```

Comment:
question: Would this `TODO`'d support mean we need to write a polyfill?

Reply:
Yeah, similar to the inverse fn above.